.github/workflows/builder_bazel_slsa3.yml

```diff
@@ -35,6 +35,11 @@ on:
         required: false
         type: string
         default: ""
+      docker-image:
```

Comment:
let's not call it docker, because we can build OCI images without docker. Also, we need a digest:
https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/builder_container-based_slsa3.yml#L62-L74

Reply:
Changed. Also added input for digest with a todo for verification of it later. TODO #2630

Reply:
I might have misunderstood you. Did you mean to use the digest to verify the image after pulling?

Like: Pull Image, Get Digest of that Image, Compare with Inputted Digest

or

Like: Pull image in form `"${BUILDER_IMAGE}@${BUILDER_DIGEST}"`, that was taken from later on in the link. IIUC, They pull the image based of the digest.

Which did you mean so I can update the issue that tracks this progress accordingly?

Reply:
`docker pull "${image}@${digest}"` should verify the digest

Reply:
added.